dask_sql/physical/rex/core/call.py

```diff
@@ -929,6 +929,11 @@ def date_part(self, what, df: SeriesOrScalar):
             return df.week
         elif what in {"YEAR", "YEARS"}:
             return df.year
+        elif what == "DATE":
+            if isinstance(df, pd.Timestamp):
+                return df.date()
+            else:
+                return dd.to_datetime(df.strftime("%Y-%m-%d"))
```

Comment:
It might also make sense to check for dask-cudf todatetime support and raise here if we have an older version, mentioning the lack of support for versions older than 2023.5.1 if df is a dask-cudf dataframe

Reply:
Thanks! Updated.